src/core/functions-versions.ts

```diff
@@ -0,0 +1,69 @@
+import { DEFAULT_DOTNET_ISOLATED_VERSION, DEFAULT_DOTNET_VERSION, DEFAULT_NODE_VERSION, DEFAULT_PYTHON_VERSION } from "./constants";
+import { logger } from "./utils/logger";
+
+export function getDefaultVersion(apiLanguage: string): string {
```

Comment:
unit test for the methods ?

Reply:
will add tests for this function and other functions in next iterations.